.github/CHANGELOG.md

```diff
@@ -576,7 +576,9 @@ random_mat2 = rng.standard_normal(3, requires_grad=False)
   [(#1228)](https://github.com/XanaduAI/pennylane/pull/1228)
 
 <h3>Bug fixes</h3>
-
+* Review of new features and fix some documentation problems.
+  [(#1409)](https://github.com/XanaduAI/pennylane/pull/1409)
+  
```

Comment:
I don't think this is something that needs to be put in the changelog, if it's referring to features that are already documented elsewhere in changelog

Reply:
Ok no problem, I'll remove it